openedx/openedx-platform
# fix: if pages and resources view is disabled, show all pages in studio [BB-6338]

## Description

In a previous PR #28686, the ability to see and enable/disable wiki and progress tabs was removed from studio along with the ability to re-order non-static tabs. The ability to toggle the Wiki tab was moved to the pages and resources section of the course authoring MFE. If that MFE is unavailable this means there is no way to show/hide the Wiki. This reverts some of the oldchanges if the pages and resources view is disabled.

## Supporting information

- https://github.com/openedx/build-test-release-wg/issues/173

## Testing instructions

- Visit `Content > Pages` in Studio for a course. 
- It should display all the course tabs in the course. 
- You should be able to show/hide the wiki tab, and the progress tab. 
- You should be able to reorder static tabs / custom pages. 
- No other tabs should be sortable.
- Set up the course authoring MFE url and enable the`discussions.pages_and_resources_mfe`  waffle flag for the course or globally.
- Visit `Content > Pages` in Studio for the course for which the flag has been set. 
- It should display only static tabs / custom pages. 
- You should be able to reorder these pages.

## Deadline

ASAP

## Other information

This does **not** revert the changes made to tab moving that disallow moving non static course tabs/pages.
